src/pr-info.ts

```diff
@@ -358,23 +359,24 @@ async function getPackageInfosEtc(
 
 async function categorizeFile(path: string, contents: (oid?: string) => Promise<string | undefined>): Promise<[string|null, FileInfo]> {
     // https://regex101.com/r/eFvtrz/1
-    const match = /^types\/(.*?)\/.*?[^\/](?:\.(d\.ts|tsx?|md))?$/.exec(path);
+    const match = /^types\/(.*?)\/.*?[^/](?:\.(d\.ts|tsx?|md))?$/.exec(path);
```

Comment:
(I'm not sure about this, it seems like consistent escaping of `/` has a point too.)

Reply:
Restored :heavy_check_mark: